DecipherNow/gm-fabric-dashboard
# chore: set CircleCI timezone to EST before running tests

Resolves #1483 

## Proposed Changes

  - Set timezone in CircleCI Config
  - Also explicitly set the TZ environment variable immediately before running `npm test` to force POSIX behavior between Mac and Linux (this sacrifices clean Windows support pending use of `moment` or `date-fns`
  
## How to Test

  - Confirm that `npm test` passes locally without snapshot errors
  - Confirm that CircleCI passed without snapshot errors
  - `npm start` and confirm the Summary components don't have visual regressions
